.github/workflows/lint.yaml

```diff
@@ -71,7 +91,14 @@ jobs:
         run: pnpm install --frozen-lockfile
 
       - name: Run Prettier
-        run: pnpm lint:prettier
+        run: |
+          if [ "${{ github.event_name }}" = "pull_request" ]; then
+            pnpm lint:prettier:check-pr
+          else
+            pnpm lint:prettier
+          fi
+        env:
+          GITHUB_BASE_REF: ${{ github.base_ref }}
```

Comment:
No need for this distinction, always use the `check-pr` target inside a workflow